install.sh

```diff
@@ -449,6 +449,7 @@ else
         exit 1
     else
 	sudo patch /usr/local/bin/vsm-controller < vsm-controller.patch
```

Comment:
Why patch vsm-controller in place? Why not just change the source version in the vsm project so it's deployed with the correct code?